Utility.cpp

```diff
@@ -1,15 +1,24 @@
+#include <cassert>
+
 #include "Utility.h"
 
 #include "HelpfulItem.h"
 #include "DefensiveItem.h"
+#include "AttackItem.h"
 #include "Character.h"
 
+#include "Dwarf.h"
+#include "Paladin.h"
+#include "Dragon.h"
+#include "DragonSlayer.h"
+
+struct Item;
 
 std::vector<std::unique_ptr<Item>> makeHelpfulItems(int num)
 {
     std::vector<std::unique_ptr<Item>> items;
     
-    while( num-- >= 0 )
+    while( num-- > 0 )
```

Comment:
This did not need to be changed.
Don't change code that doesn't need changing.
- [ ] revert